data/Debian.yaml

```diff
@@ -1,2 +1,19 @@
 ---
 prometheus::env_file_path: '/etc/default'
+prometheus::bin_dir: '/usr/bin'
+prometheus::shared_dir: '/usr/share/prometheus'
+prometheus::install_method: 'package'
```

Comment:
@anarcat Why do change the install method for Debian? It makes this pull request backwards incompatible and can be set by yourself, if necessary.

Reply:
i'm trying to reduce the diff with my local stuff. but if that would help get this through, sure, i can revert this...

Reply:
so i tried reverting this part, but it also involved removing a bunch more configuration, as you can see in 2e70883 - it doesn't feel right...